File: frontend/appGestionDesTaches/src/app/interface/task/task-create-dto.ts
import { EPeriorite } from "../../enums/e-periorite";

export interface TaskCreateDTO {
  titre: string;
  description: string;
  idParentTask?: number | null;  
  idDeveloppeur?: number | null;
  idProjet?: number | null;
  progress: number;
  periorite: EPeriorite;
}

File: backend/gestionDestaches/src/main/java/org/example/gestiondestaches/DTO/TaskDTO/TaskCreateDTO.java
package org.example.gestiondestaches.DTO.TaskDTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.example.gestiondestaches.entite.Task;
import org.example.gestiondestaches.enummiration.EPeriorite;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TaskCreateDTO {
    private String titre;
    private String description;
    private Long idParentTask;
    private  Long idDeveloppeur;
    private Long idProjet;
    private int progress;
    private EPeriorite periorite;


}
